test/modules/TestResultCoordinator/TestMode.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator
+{
+    enum TestMode
```

Comment:
maybe it would make sense to add some comments on what is the difference b/w these two modes.